src/Autofac/Core/Activators/Reflection/ReflectionActivator.cs

```diff
@@ -55,18 +55,21 @@ public class ReflectionActivator : InstanceActivator, IInstanceActivator
             Type implementationType,
             IConstructorFinder constructorFinder,
             IConstructorSelector constructorSelector,
+            IPropertyFinder propertyFinder,
```

Comment:
I wasn't sure if I should add this here or as another overload. Since this is a public constructor on a public type, it will break anything that has taken a dependency on it. I added here for now (since 4.0 hasn't been finalized), but can create an overload if preferred.